### PR TITLE
feat(macros/CSSRef): Add support for “Grouping Selectors”

### PR DESCRIPTION
**Pull request by ExE-Boss**
This is essentially #942, but for mdn/sprints#2233.

---

review?(@chrisdavidmills, @davidflanagan, @Elchi3, @escattone, @peterbe)

### CODE REVIEW

File: macros/CSSRef.ejs [hunk 1]
```diff
@@ -31,9 +31,10 @@ if (slug) {
         'Guides': mdn.getLocalString(commonl10n, 'Guides'),
         'tag_selectors':  mdn.getLocalString(cssl10n, 'tag_selectors'),
         'Basic Selectors': mdn.getLocalString(cssl10n, 'Basic Selectors'),
+        'Combinators': mdn.getLocalString(cssl10n, 'Combinators'),
+        'Grouping Selectors': mdn.getLocalString(cssl10n, 'Grouping Selectors'),
```
[the remark this answers is not in the document]
Comment (ramiy) [reply macros/CSSRef.ejs]: And replace `Grouping Selectors` with simple `Grouping`.

Ref:
https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors#Grouping
https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/Reference#Grouping

[the remark this answers is not in the document]
Comment (ExE-Boss) [reply macros/CSSRef.ejs]: This is&nbsp;the&nbsp;localisation code, and&nbsp;this&nbsp;has&nbsp;to&nbsp;match what’s&nbsp;in&nbsp;`mdn‑data`: https://github.com/mdn/data/blob/99a0ffcb2e5f317af65ab23cd7d6d6de683d89f2/css/definitions.json#L63

Also, the&nbsp;order here&nbsp;doesn’t&nbsp;affect the&nbsp;generated&nbsp;sidebar.

**Review by ramiy — APPROVED** [review]

**Comment by ramiy** [comment]
@ExE-Boss Any updates?

**Comment by ExE-Boss** [comment]
@ramiy This&nbsp;needs&nbsp;someone from&nbsp;@MDN to&nbsp;merge&nbsp;this.

**Comment by chrisdavidmills** [comment]
Can I please request that someone test this and then merge if it looks OK? @Elchi3 maybe? ;-)

**Comment by Elchi3** [comment]
How important is this? The CSS sidebar is fragile and I don't want to invest unnecessary time since KumaScript is on its way to the graveyard.

**Comment by chrisdavidmills** [comment]
It's probably not that important. I'd be OK with leaving this to fix in Stumptown.

**Comment by ramiy** [comment]
@Elchi3 What is the alternative to KumaScript, and when are do you expect to deprecate it?

**Comment by Elchi3** [comment]
The alternative is to built new sidebars that can be changed much more easily. 
I expect KumaScript to not survive 2020.

**Comment by ramiy** [comment]
How can I build a new sidebars? Do you have examples? Or other pages that have similar sidebars without KumaScript?

**Comment by Elchi3** [comment]
We don't have renewed sidebars yet, but we've started looking into it.

**Comment by ramiy** [comment]
@Elchi3 if there is no alternative and there is no timeline when an alternative will be available, I think this PR should be merged.

**Comment by ExE-Boss** [comment]
I&nbsp;agree, let’s&nbsp;merge&nbsp;this&nbsp;now, and&nbsp;worry&nbsp;about implementing&nbsp;<https://discourse.mozilla.org/t/css-sidebar-proposal/49500>&nbsp;later.

**Comment by Elchi3** [comment]
There is no timeline yet, but our energy and limited time is spent on renewed sidebars and not making the old fragile sidebars even more complicated. 
See https://discourse.mozilla.org/t/css-sidebar-proposal/49500 for how the current situation is not great for us and a proposal for how we want to proceed.

**Pull request closed without merging**